mysql-test/t/rotate_system_key_check_privileges.test

```diff
@@ -0,0 +1,15 @@
+--source include/not_embedded.inc
```

Comment:
Please prefix testcase with percona_ (easier to tell apart from upstream testcases, and not required in fully our *_encryption suites)

Reply:
Done